aep/general/0148/aep.md.j2

```diff
@@ -1,5 +1,155 @@
 # Standard fields
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/28) for more
-information.
+Certain concepts are common throughout any corpus of APIs. In these situations,
+it is useful to have a standard field name and behavior that is used
+consistently to communicate that concept.
+
+## Guidance
+
+Standard fields **should** be used to describe their corresponding concept, and
+**should not** be used for any other purpose.
+
+### Resource paths and IDs
+
+#### path
+
+Every resource **must** have a `string path` field, used for the resource path
+(AEP-122), which **should** be the first field in the resource.
+
+#### parent
+
+The `string parent` field refers to the resource name of the parent of a
+collection, and **should** be used in most `List` (AEP-132) and `Create`
```

Comment:
shouldn't this be a must?

Reply:
Top-level resources are a thing - but I updated this to say "**must**...except".